docs/lang/articles/basic/external.md

```diff
@@ -9,7 +9,8 @@ efficiently manipulating Taichi field data in Python-scope could also be
 helpful.
 
 We provide various interfaces to copy the data between Taichi fields and
-external arrays. The most typical case maybe copying between Tachi
+external arrays. External arrays refer to NumPy arrays or PyTorch tensors.
+The most typical case may be copying between Tachi
 fields and Numpy arrays. Let's take a look at two examples below.
 
 **Export data in Taichi fields to a NumPy array** via `to_numpy()`. This
```

Comment:
A few nits. As I cannot insert suggested changes for unmodified lines so I put the suggested version here:
````md
- **Export data in Taichi fields to NumPy arrays** via `to_numpy()`. This
allows us to export computation results to other Python packages that
support NumPy, e.g. `matplotlib`:

```python {8}
@ti.kernel
def my_kernel():
   for i in x:
      x[i] = i * 2
x = ti.field(ti.f32, 4)
my_kernel()
x_np = x.to_numpy()
print(x_np)  # np.array([0, 2, 4, 6])
```

- **Import data from NumPy arrays to Taichi fields** via `from_numpy()`.
This allows us to initialize Taichi fields via NumPy arrays:
````